{{cookiecutter.repo_root}}/hack/verify-manifests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+main() {
+
+    "${SCRIPT_DIR}"/build-manifests.sh
+    if [[ $(git status --porcelain) ]]; then
+        git diff --exit-code >&2 || {
+            echo "Did you forget to build the manifests locally?" >&2;
+            echo "Please run ./hack/build-manifests.sh and update your PR" >&2;
+            echo "Or run ./hack/generate-everything.sh to run all the generators at once." >&2;
```

Comment:
As you mentioned offline, we should probably remove the line about `/hack/generate-everything.sh` for now

Reply:
removed the line